docs/Conditions.md

```diff
@@ -191,4 +211,50 @@ Usage can then look as following:
 #else
 // This renders for desktop platforms
 #endif
-```
\ No newline at end of file
+```
+
+## Conditional Parameters
+
+[Parameter symbols in template](Reference-for-template.json.md#parameter-symbol) can be specified together with optional conditions:
+* [`IsEnabled Condition`](Reference-for-template.json.md#isEnabled) - overwritting presence of input parameter. If condition is specified and evaluates to false (or a `false` constant is passed), passed parameter value (if any) is ignored and processing works as if the parameter value was not passed. This includes application of [default values](Reference-for-template.json.md#default), [verification of mandatory parameters](Reference-for-template.json.md#isRequired), [conditional processing of sources](Conditional-processing-and-comment-syntax.md) and [replacements](Reference-for-template.json.md#replaces).
```

Comment:
> overwritting presence of input parameter

I'm not sure what you mean to say with this sentence. I might suggest something like:

> Used to determine when (or if) this symbol should be used. If this condition is specified and evaluates to `false`, then this parameter is treated as if it does not exist. This applies to the use of [default values](Reference-for-template.json.md#default), [verification of required parameters](Reference-for-template.json.md#isRequired), [conditional processing of sources](Conditional-processing-and-comment-syntax.md) and [replacements](Reference-for-template.json.md#replaces)